server/server.js

```diff
@@ -46,6 +51,76 @@ app.get('/songDescription/:songId', async(req, res) => {
   }
 });
 
+app.post('/songDescription', async(req, res) => {
+  // console.log(req.body);
+  try {
+    const description = await db.saveDescriptions(req.body);
+    res.status(200).json({
+      success: true,
+      msg: 'create one song description',
```

Comment:
Consider having this refer to the `songId` (akin to line 81) if it exists on the `req.body` object.